web/includes/database.php

```diff
@@ -55,6 +55,7 @@ function dbConnect() {
         PDO::MYSQL_ATTR_SSL_CA   => ZM_DB_SSL_CA_CERT,
         PDO::MYSQL_ATTR_SSL_KEY  => ZM_DB_SSL_CLIENT_KEY,
         PDO::MYSQL_ATTR_SSL_CERT => ZM_DB_SSL_CLIENT_CERT,
+        PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => ZM_DB_SSL_VERIFY_SERVER_CERT,
```

Comment:
My concern here is upgrades, people's existing files in /etc/zm.conf won't have the define, so this will break, so we need to ensure that ZM_DB_SSL_VERIFY_SERVER_CERT gets defined no matter what.